rmw_fastrtps_shared_cpp/src/utils.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "rmw_fastrtps_shared_cpp/utils.hpp"
+
+#include "rmw/rmw.h"
+#include "fastrtps/types/TypesBase.h"
```

Comment:
```suggestion
#include "fastdds/dds/topic/Topic.hpp"
#include "fastrtps/types/TypesBase.h"
```

`TopicQos` and `TopicDescription` are already included in the `utils.hpp` header file, but `Topic` is also used and its header file is not being included.